packages/gluestick/src/renderer/middleware.js

```diff
@@ -24,75 +22,83 @@ const errorHandler = require('./helpers/errorHandler');
 const getCacheManager = require('./helpers/cacheManager');
 const getStatusCode = require('./response/getStatusCode');
 const createPluginUtils = require('../plugins/utils');
+const hooksHelper = require('./helpers/hooks');
+
+const entries = require('project-entries').default;
```

Comment:
Many of these will be removed on the next ticket, as they're just passed along to `render.js`.